apps/api/src/app/shared/dtos/pagination-request.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiPropertyOptional } from '@nestjs/swagger';
+
+export class PaginationRequestDto {
+  @ApiPropertyOptional({
+    type: Number,
+    required: false,
+  })
+  page?: number = 0;
+
+  @ApiPropertyOptional({
+    type: Number,
+    required: false,
+    default: 10,
+    maximum: 100,
```

Comment:
One problem with this approach is that different pagination endpoints might have different default and maximum values. We must think in a way to provide those values dynamically.

Reply:
Done